rosdep/python.yaml

```diff
@@ -2239,6 +2239,11 @@ python-ipdb:
   debian: [python-ipdb]
   gentoo: [dev-python/ipdb]
   ubuntu: [python-ipdb]
+python-is-python3:
+  ubuntu: 
```

Comment:
There is trailing whitespace on this line, which is why CI is unhappy.

Reply:
Removed the whitespace. Thanks for the hint!